core/unit_test/TestViewAPI.hpp

```diff
@@ -841,6 +841,24 @@ struct TestViewMirror {
     ASSERT_EQ(a_org(5), a_h3(5));
   }
 
+  template <typename View>
+  static typename View::const_type view_const_cast(View const &v) {
+    return v;
+  }
+
+  static void test_mirror_copy_const_data_type() {
+    using ExecutionSpace = typename DeviceType::execution_space;
+    int const N          = 100;
+    Kokkos::View<int *, ExecutionSpace> v("v", N);
+    Kokkos::deep_copy(v, 255);
+    auto v_m1 = Kokkos::create_mirror_view_and_copy(
+        Kokkos::DefaultHostExecutionSpace(), view_const_cast(v));
+    ASSERT_TRUE(run_check(v_m1, 255));
```

Comment:
I moved it from `core/unit_test/TestViewCopy.hpp` and of course `run_check()` is not defined there